frontstage/views/surveys.py

```diff
@@ -240,10 +240,13 @@ def upload_survey(session):
         logger.debug('Upload successful')
         return render_template('surveys/surveys-upload-success.html', _theme='default', upload_filename=upload_filename)
     else:
-        logger.debug('Upload failed')
-        if result.text is None:
+        logger.error('Upload failed. url: {}, status code: {}'.format(url, result.status_code))
```

Comment:
We're probably going to change the formatting of logs in an upcoming card maybe but the content here all looks good